cirq-core/cirq/__init__.py

```diff
@@ -679,3 +680,5 @@
 from cirq import contrib
 
 # pylint: enable=wrong-import-position
+
```

Comment:
If you run the formatter (or download "black" and run it) it should clear most of this formatting.